src/Application/View/Extension/DocumentExtension.php

```diff
@@ -72,7 +72,7 @@ public function getDocumentIcon(string $filename): string
 
         $extension = strtolower(pathinfo($filename, PATHINFO_EXTENSION));
 
-        if (static::$extensionIcon === null) {
+        if ($this->extensionIcon === null) {
```

Comment:
`extensionIcon` is a static variable.
What was the PHPStan error?

Reply:
```
Unsafe access to private property Glpi\Application\View\Extension\DocumentExtension::$extensionIcon through static::.
```